Refresh.GameServer/Types/Levels/Tags.cs

```diff
@@ -0,0 +1,193 @@
+using System.Text;
+
+namespace Refresh.GameServer.Types.Levels;
+
+public enum Tags
```

Comment:
This needs to be renamed to `Tags`; enums should never be plural. Especially when they're not flags.

Could also be a `byte`.

Reply:
i couldnt point a finger on why the name felt so wrong to say/type, will change to `Tag`, and make it a `byte`